contracts/GenArt721CoreV3.sol

```diff
@@ -32,6 +32,17 @@ contract GenArt721CoreV3 is ERC721, Ownable, IGenArt721CoreContractV3 {
     uint256 constant ONE_MILLION = 1_000_000;
     uint256 constant FOUR_WEEKS_IN_SECONDS = 2_419_200;
 
+    /// Art Blocks previous flagship token addresses (for reference)
+    address public constant ART_BLOCKS_TOKEN_ADDRESS_0 =
+        0x059EDD72Cd353dF5106D2B9cC5ab83a52287aC3a;
+    address public constant ART_BLOCKS_TOKEN_ADDRESS_1 =
+        0xa7d8d9ef8D8Ce8992Df33D8b8CF4Aebabd5bD270;
```

Comment:
One minor thing: before deployment we should add in documentation comments the project ID ranges for each historic token address.

Also naming nit, WDYT about `ART_BLOCKS_ERC721TOKEN_ADDRESS_V0`/`_V1` nomenclature?

Reply:
yes and yes 👍 end range for V1 contract currently flagged as a TODO comment